docs/cluster-guide.md

```diff
@@ -0,0 +1,84 @@
+# Guide to setting up a 3-node cluster
+
+Welcome to the quick start guide for setting up a 3-node cluster using EventStoreDB. This guide covers how to prepare the environment, generate the required certificates, and how to configure and run the cluster.
+
+## Preparing the environment
+
+1. Create a folder named 'Cluster' on the desktop. 
+    - For production use, place the 'Cluster' folder in the root directory (C:).
```

Comment:
```suggestion
```

Reply:
I'd never put any database file in the volume containing the OS in a production scenario @cdevarenne